src/main/scala/sigmastate/serialization/BigIntConstantSerializer.scala

```diff
@@ -0,0 +1,37 @@
+package sigmastate.serialization
+
+import java.math.BigInteger
+
+import com.google.common.primitives.{Ints, Shorts}
+import sigmastate.SBigInt
+import sigmastate.SType.TypeCode
+import sigmastate.Values.BigIntConstant
+import sigmastate.serialization.ValueSerializer._
+
+object BigIntConstantSerializer extends ValueSerializer[BigIntConstant] {
+
+  val lengthSize: Int = 2
```

Comment:
There is convention that constants should have upper camel case `LengthSize`.
[Convention Doc](https://docs.scala-lang.org/style/naming-conventions.html)